src/i18n/l10n/en.ftl

```diff
@@ -731,6 +731,7 @@ MONTHS = MONTHS
 # Label for section of ObsDetails with information about observation projects, DQA, etc.
 More = More
 More-info = More info
+More-ways-to-observe = More ways to observe
```

Comment:
This could use a comment for translators, something like `# Header for pivot card explaining long press functionality to users`